src/app/globals.css

```
@@ -104,6 +104,18 @@ a {
   50% { transform: scale(1.02); opacity: 1; }
 }
 
+/* Current pathway node — gentle amber glow (#336) */
+@keyframes pathwayPulse {
```

Stylelint (17.13.0)

[error] Expected keyframe name "pathwayPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)